components/docs/NavSideBar.vue

```diff
@@ -51,6 +52,15 @@
                 type: Object,
             },
         },
+        data() {
+          return {
+              disabledPages: [
+                  '/docs/terraform/data-sources',
+                  '/docs/terraform/guides',
+                  '/docs/terraform/resources'
+              ]
```

Comment:
Please can you add some screenshots of why you have to do that ? I don't really like the fact that we have arbitrary content-related things in a generic component

Reply:
https://github.com/kestra-io/docs/issues/815
for those pages we don't have content and we need to disable that pages